### PR TITLE
Restyle resources page with new schema

### PR DESCRIPTION
**Pull request by ericjim**
### Changes
* Make a new resources page

### Preview
* https://new-schema.firebaseapp.com/resources/

@StephenFluin

### CODE REVIEW

File: public/_data.json [hunk 1]
```diff
@@ -31,21 +31,5 @@
 
   "presskit": {
     "title": "Press Kit"
-  },
```
Comment (ericjim) [inline public/_data.json]: Originally we believed these would be separate pages, eventually we consolidated everything. So these pages are no longer needed.

**Comment by naomiblack** [comment]
I'll need a bunch of data populated before I'm happy merging to master. Right now most of the entries show up as _No Description_

**Comment by StephenFluin** [comment]
@naomiblack I can have that in tonight. Plan on merging it Thursday?

**Comment by ericjim** [comment]
@StephenFluin thanks for pointing that out, I'll fix it.

**Comment by ericjim** [comment]
@StephenFluin I've made an update, you can see it here now https://new-schema.firebaseapp.com/resources/ , make sure to do a hard refresh. I don't know what to do with the nav in smaller screens. So I am hiding it when the screen real estate gets tight. We can revisit the design later, or we can gather suggestions.

**Comment by StephenFluin** [comment]
@ericjim This fixes the usability and think it's good enough. We can revisit further. @naomiblack content is in, so this might be ready to merge

**Pull request merged**